lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -439,9 +439,10 @@ class ConvertTorchToArith
     target.addIllegalOp<Torch::ConstantIntOp>();
     patterns.add<ConvertTorchConstantIntOp>(typeConverter, context);
 
-    target.addIllegalOp<AtenIntBoolOp, AtenFloatScalarOp>();
+    target.addIllegalOp<AtenIntBoolOp, AtenFloatScalarOp, AtenIntScalarOp>();
```

Comment:
Why this change? just curious.

Reply:
I was following the examples of the `AtenIntBoolOp` and `AtenFloatScalarOp` (they were also marked as illegal op and a pattern was added for each).

Reply:
Yeah torch-mlir does a partial conversion in each of its parts, so each lowering needs to mark the ops it's supposed to lower as illegal. It mainly helps report an error in the right place.